devsetup/scripts/edpm-compute-repos.sh

```diff
@@ -31,6 +31,7 @@ if [[ ! -f $SSH_KEY ]]; then
     exit 1
 fi
 
+if [[ ! -f $CMDS_FILE ]]; then
```

Comment:
The changes looks good. Can we set a default empty value for `CMDS_FILE` at Line 27? Something like that:
```
CMDS_FILE=${CMDS_FILE:-""}
```

Reply:
+1, we need to default the var

Reply:
@raukadah there is a default value already in line 25: `CMDS_FILE=${CMDS_FILE:-"/tmp/edpm_compute_repos"}`
if we change valut to be empty, it will break the current deployment, since nobody sets this file path. This is a temporary file that is copied inside the compute instance and gets deleted afterwards.

Reply:
Oh sorry I missed that var. thank you for pointing that out.